packages/lit-dev-content/site/tutorials/content/custom-attribute-converter/01.md

```diff
@@ -0,0 +1,78 @@
+In this example:
+
+- `my-element` is a Lit component that:
+  - Has a `date` reactive property of type `Date`
+  - Has attribute to property conversion turned off `{attribute: false}`
+  - Displays out the given `date` in a human-readable-format
+- `index.html` passes the the date `05/05/22` to `my-element` as a `Date` object via property
+
+<aside class="info">
+
+The reason we have turned off attribute conversion is because the `@property` decorator or `static properties` block will tell Lit to convert a string attribute to the type passed in as the `type` option, but Lit does not know how to handle `type: Date` (string attribute to JS `Date` object conversion). This means that the only way to pass `date` to `my-element` is via Javascript which may lead to a bad developer experience.
+
+</aside>
+
+## Attribute Converters
+
+Attribute converters tell Lit how to convert an attribute to a property and, in the case that a reactive property has the `reflect: true` flag, vice versa.
+
+### Built-in Attribute Converters
+
+Lit has several built-in attribute converters. You can choose which built-in converter is invoked by setting the `type` property of the [reactive property options](/docs/components/properties/#property-options). The built-in converters are:
+
+* `String` *(default)*
+* `Boolean`
+* `Number`
+* `Array`
+* `Object`
+
+Notice that Lit does not have a `Date` converter built-in and thus you cannot do `type: Date` to accept a date-string attribute. You can read more on how built-in coverters convert attributes to properties and vice versa in the [Lit documentation](/docs/components/properties/#conversion-type).
+
+## Goal for this step
+
+`my-element` accepts human-readable date attribute as a reactive property. e.g. `date-str="05/05/22"`
+
+<aside class="negative">
+
+This section is meant to exemplify the need for custom attribute converters and
+is not recommended best-practice code. If you'd like to skip to the section in
+which you write best-practice code, go to [step 4](#03).
+
+</aside>
```

Comment:
What do you think about moving this above the Goal section? And say "This and the following section are meant to ..." ?